xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -738,7 +738,7 @@ bool CGUIWindowVideoBase::OnItemInfo(int iItem)
 
   OnItemInfo(item.get(), scraper);
 
-  return true;
+  return item->HasVideoInfoTag();
```

Comment:
Thought about this as well but wanted to keep the changes to a minimum. Will check and change it.